gradle/libs.versions.toml

```diff
@@ -64,7 +64,7 @@ protobuf = "3.22.3"
 paparazzi = "1.0.0"
 paparazziNative = "2022.1.1-canary-f5f9f71"
 shadow = "8.1.1"
-skiko = "0.9.30"
```

Comment:
> Release Notes

Please update the release notes, collecting fixes from the Skiko commits

Reply:
There wasn't a ticket for every change. If we enumerate them, we need to create tickets in SKIKO (not in COMPOSE, to avoid misunderstandings where it is fixed)